.clj-kondo/config.edn

```diff
@@ -1 +1,3 @@
-{:config-paths ["../resources/clj-kondo.exports/clj-commons/aleph"]}
+{:config-paths ["../resources/clj-kondo.exports/clj-commons/aleph"]
+ :linters      {:unsorted-required-namespaces {:level :warning}}
+ :lint-as      {manifold.deferred/let-flow clojure.core/let}}
```

Comment:
Ideally this would be provided by `manifold`, of course. There are a few more like this coming up. Right now I tend to collect all of them here and once everything is green, extract them out and file PRs against the other projects. Sound good?

Reply:
That's fine.

`let-flow'` should be linted the same way.